README.md conan commands throw errors

**Describe the bug**
I'm new to conan and entered the build commands as described in the README.md but conan seems to have changed its syntax and throws errors.

**To Reproduce**

As specified in README.md: (after installing conan with pip)

```
rdf4cpp$ conan user                                                                                                                                                                                                                                                                                                   
'user' is not a Conan command. See 'conan --help'.
ERROR: Unknown command 'user'
rdf4cpp$ conan profile new --detect default                                                                                                                                                                                                                                                                           
usage: conan profile [-h] [-v [V]] {detect,list,path,show} ...
conan profile: error: argument subcommand: invalid choice: 'new' (choose from 'detect', 'list', 'path', 'show')
ERROR: Exiting with code: 2
conan profile update settings.compiler.libcxx=libstdc++11 default
rdf4cpp$ conan profile update settings.compiler.libcxx=libstdc++11 default                                                                                                                                                                                                                                            develop
usage: conan profile [-h] [-v [V]] {detect,list,path,show} ...
conan profile: error: argument subcommand: invalid choice: 'update' (choose from 'detect', 'list', 'path', 'show')
ERROR: Exiting with code: 2
rdf4cpp$ conan remote add dice-group https://conan.dice-research.org/artifactory/api/conan/tentris

rdf4cpp$ cmake -B build                                                                                                                                                                                                                                                                                               develop
-- Conan: checking conan executable
-- Conan: Found program /home/konrad/.local/bin/conan
-- Conan: Version found Conan version 2.0.12
-- Conan executing: /home/konrad/.local/bin/conan install /home/konrad/tmp/rdf4cpp/conanfile.py --build missing --env:host CC=/usr/bin/cc --env:host CXX=/usr/bin/c++ --options with_test_deps=False --options boost:header_only=True --settings build_type=Release --settings compiler=gcc --settings compiler.version=13 --settings compiler.libcxx=libstdc++11
usage: conan install [-h] [-v [V]] [-f FORMAT] [--name NAME]
                     [--version VERSION] [--user USER] [--channel CHANNEL]
                     [--requires REQUIRES] [--tool-requires TOOL_REQUIRES]
                     [-b BUILD] [-r REMOTE | -nr] [-u] [-o OPTIONS_HOST]
                     [-o:b OPTIONS_BUILD] [-o:h OPTIONS_HOST]
                     [-pr PROFILE_HOST] [-pr:b PROFILE_BUILD]
                     [-pr:h PROFILE_HOST] [-s SETTINGS_HOST]
                     [-s:b SETTINGS_BUILD] [-s:h SETTINGS_HOST] [-c CONF_HOST]
                     [-c:b CONF_BUILD] [-c:h CONF_HOST] [-l LOCKFILE]
                     [--lockfile-partial] [--lockfile-out LOCKFILE_OUT]
                     [--lockfile-packages] [--lockfile-clean]
                     [--lockfile-overrides LOCKFILE_OVERRIDES] [-g GENERATOR]
                     [-of OUTPUT_FOLDER] [-d DEPLOYER]
                     [--deployer-folder DEPLOYER_FOLDER] [--build-require]
                     [path]
conan install: error: unrecognized arguments: --env:host CC=/usr/bin/cc --env:host CXX=/usr/bin/c++
ERROR: Exiting with code: 2
CMake Error at build/conan.cmake:651 (message):
  Conan install failed='2'
Call Stack (most recent call first):
  cmake/conan_cmake.cmake:18 (conan_cmake_install)
  CMakeLists.txt:16 (install_packages_via_conan)
```

**Environment**
 - OS: Arch Linux
 - Conan: 2.0.12
 - gcc 13.2.1
 - cmake 3.27.5

--- a/README.md
+++ b/README.md
@@ -1,4 +1,6 @@
 ⚠️ This repo is work-in-progress! Before v0.1.0 all APIs are considered unstable and might be subject to change. ⚠️
+⚠️ rdf4cpp uses Conan 1 and is not compatible with Conan 2. ⚠️
+
 # rdf4cpp
 
 rdf4cpp aims to be a stable, modern RDF library for C++.
@@ -47,10 +49,10 @@ CI builds and tests rdf4cpp with gcc-{11,12,13}, clang-{14,15,16} with libstdc++
 
 ### Dependencies
 
-It is recommended to include build dependencies via conan. Set up Conan as follows on Ubuntu 22.04+:
+It is recommended to include build dependencies via conan version 1. Set up Conan as follows on Ubuntu 22.04+:
 ```shell
 sudo apt install python3-pip
-pip3 install --user conan
+pip3 install --user "conan<2"
 conan user
 conan profile new --detect default
 conan profile update settings.compiler.libcxx=libstdc++11 default

